google-cloud-vision/.rubocop.yml

```diff
@@ -29,16 +30,7 @@ Metrics/ClassLength:
   Enabled: false
 Metrics/MethodLength:
   Max: 26
-  Exclude:
-    - "lib/google/cloud/vision.rb"
 Metrics/ParameterLists:
   Enabled: false
-Layout/EmptyLineAfterGuardClause:
-  Enabled: false
 Layout/EmptyLines: # for the extra line between copyright and code
   Enabled: false
-Naming/FileName:
-  Exclude:
-    - "lib/google-cloud-vision.rb"
-Naming/UncommunicativeMethodParamName:
-  Enabled: false
```

Comment:
With rubocop 0.59.2, you'll probably want to re-add these 2 line.

Reply:
Thanks, will do.